app/controllers/admin/workshops_controller.rb

```diff
@@ -122,12 +122,11 @@ def workshop_id
 
   def set_host(host_id)
     return unless host_id
-
     host = @workshop.workshop_sponsors.find_or_initialize_by(sponsor_id: host_id)
-    unless @workshop.host.eql?(host.sponsor)
-      @workshop.workshop_sponsors.where(sponsor: @workshop.host).destroy_all
-      host.update(host: true)
-    end
+    return if @workshop.host.eql?(host.sponsor)
```

Comment:
@notapatch this doesn't seem correct to me, shouldn't this be `return unless @workshop.host.eql?(host.sponsor)`? E.g. you only want this block code to run if the sponsor is also a host.

Reply:
there's a rebase issue here ... this should only be the "Freeze mutable constants code"